Lunch/ViewController.swift

```diff
@@ -8,11 +8,18 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UITextFieldDelegate {
+
+    @IBOutlet weak var lunchTableView: UITableView!
+    @IBOutlet weak var lunchTextField: UITextField!
+
     var lunchChoice: Int!
-    
-    override func viewDidLoad() {
+    var items: [String] = []
+
+        override func viewDidLoad() {
         super.viewDidLoad()
+        self.lunchTableView.registerClass(UITableViewCell.self, forCellReuseIdentifier: "LunchItemCell")
```

Comment:
I always make constants for `String` parameters like this, because it's really really easy to forget to change them if they show up in more than one place